src/mono/mono/mini/mini-llvm.c

```diff
@@ -11023,6 +11023,23 @@ process_bb (EmitContext *ctx, MonoBasicBlock *bb)
 			values [ins->dreg] = result;
 			break;
 		}
+		case OP_ARM64_SADDV:
```

Comment:
Might be easier to add just one opcode similar to the OP_XOP opcodes and stored the llvm intrinsics id in ins->inst_c0 like its done in other case.